### PR TITLE
Fix sway templates

### PR DESCRIPTION
**Pull request by xunilrj**
## Checklist

- [ ] I have linked to any relevant issues.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation where relevant (API docs, the reference, and the Sway book).
   - [ ] If my change requires substantial documentation changes, I have [requested support from the DevRel team](https://github.com/FuelLabs/devrel-requests/issues/new/choose)
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added (or requested a maintainer to add) the necessary `Breaking*` or `New Feature` labels where relevant.
- [ ] I have done my best to ensure that my PR adheres to [the Fuel Labs Code Review Standards](https://github.com/FuelLabs/rfcs/blob/master/text/code-standards/external-contributors.md).
- [ ] I have requested a review from the relevant team or maintainers.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -146,7 +146,7 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-      
+
```
Comment (JoshuaBatty) [inline .github/workflows/ci.yml:149]: We can probably remove this empty new line...

**Review by JoshuaBatty — APPROVED** [review]

File: templates/sway-predicate-test-rs/template/Cargo.toml [hunk 2]
```diff
@@ -7,7 +7,7 @@ authors = ["{{authors}}"]
 license = "Apache-2.0"
 
 [dev-dependencies]
-fuels = { version = "0.66", features = ["fuel-core-lib"] }
+fuels = { version = "0.66.1", features = [] }
```
Comment (JoshuaBatty) [inline templates/sway-predicate-test-rs/template/Cargo.toml:10]:
```suggestion
fuels = "0.66.1"
```

File: templates/sway-script-test-rs/template/Cargo.toml [hunk 3]
```diff
@@ -7,7 +7,7 @@ authors = ["{{authors}}"]
 license = "Apache-2.0"
 
 [dev-dependencies]
-fuels = { version = "0.66", features = ["fuel-core-lib"] }
+fuels = { version = "0.66.1", features = [] }
```
Comment (JoshuaBatty) [inline templates/sway-script-test-rs/template/Cargo.toml:10]:
```suggestion
fuels = "0.66.1"
```

File: templates/sway-test-rs/template/Cargo.toml [hunk 4]
```diff
@@ -7,7 +7,7 @@ authors = ["{{authors}}"]
 license = "Apache-2.0"
 
 [dev-dependencies]
-fuels = { version = "0.66.1", features = ["fuel-core-lib"] }
+fuels = { version = "0.66.1", features = [] }
```
Comment (JoshuaBatty) [inline templates/sway-test-rs/template/Cargo.toml:10]:
```suggestion
fuels = "0.66.1"
```

File: templates/sway-predicate-test-rs/template/Cargo.toml (hunk 2 above)
Comment (JoshuaBatty) [reply templates/sway-predicate-test-rs/template/Cargo.toml:10]: are we able to do this or is there a reason we want an empty features array?

Comment (xunilrj) [reply templates/sway-predicate-test-rs/template/Cargo.toml]: Done.

File: templates/sway-test-rs/template/Cargo.toml (hunk 4 above)
Comment (xunilrj) [reply templates/sway-test-rs/template/Cargo.toml]: Done.

File: templates/sway-script-test-rs/template/Cargo.toml (hunk 3 above)
Comment (xunilrj) [reply templates/sway-script-test-rs/template/Cargo.toml]: Done.

**Review by JoshuaBatty — APPROVED** [review]

**Review by IGI-111 — APPROVED** [review]

**Pull request merged**